<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
   QPushButton {
       background-color: #B22222;
       border: none;
       border-radius: 40px;
       color: white;
       font-size: 28px;
       font-weight: bold;
       padding: 10px;
   }
   QPushButton:hover {
       background-color: #cc3333;
   }
   QPushButton:pressed {
       background-color: #800000;
   }
  </string>
    </property>
    <property name="text">
     <string>‚èª</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
